Index.html

```diff
@@ -27,5 +27,113 @@ <h4 class="connect">LET'S CONNECT</h4>
       <li><img src="Assests/Vectorangerllist.svg"></li>
     </ul>
   </section>
+  <section class="worksSection">
+    <div class="space"></div>
+    <div class="firstCard">
+      <div class="snapshot"></div>
+      <h3 class="tonic">Tonic</h3>
+      <ul class="canopyList">
+        <li class="canopy">CANOPY</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="backEnd">Back End Dev</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="year">2015</li>
+      </ul>
+      <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="works">
+        <button class="htmlButton buttons"></button>
+        <button class="cssButton buttons"></button>
+        <button class="jsButton buttons"></button>
+      </div>
+      <button class="seeProject">See project</button>
+    </div>
+    <div class="space"></div>
+    <div class="space"></div>
+    <div class="firstCard">
+      <div class="snapshotTwo"></div>
+      <h3 class="stories">Multi-Post Stories</h3>
+      <ul class="canopyList">
+        <li class="canopy">CANOPY</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="backEnd">Back End Dev</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="year">2015</li>
+      </ul>
+      <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="works">
+        <button class="htmlButton buttons"></button>
+        <button class="cssButton buttons"></button>
+        <button class="jsButton buttons"></button>
+      </div>
+      <button class="seeProject">See project</button>
+    </div>
+    <div class="space"></div>
+    <div class="space"></div>
+    <div class="firstCard">
+      <div class="snapshotThree"></div>
+      <h3 class="tonic">Tonic</h3>
+      <ul class="canopyList">
+        <li class="canopy">CANOPY</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="backEnd">Back End Dev</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="year">2015</li>
+      </ul>
+      <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
[OPTIONAL]
- [ ] please make sure to use the correct font color to style this paragraph it is important to follow the design patterns provided to you on figma :+1: 
- figma design
![f-c-ff](https://user-images.githubusercontent.com/89055502/235336675-c5034e0b-b361-47ab-ad60-fe549c241d99.png)
- your design
![f-c-f](https://user-images.githubusercontent.com/89055502/235336679-10dd7556-2db3-4d33-82e8-d8e481f709b4.png)